### DISCUSSION TITLE
Unassigned faces not displayed

### DISCUSSION THREAD

**Discussion by rrrnld** [nextcloud/recognize#1114] [Q&A]
When looking at the unassigned faces (Photos → People → Scroll Down, Click Unassigned Faces), for me it looks like this:
![image](https://github.com/nextcloud/recognize/assets/525780/63e4a801-0ba2-4ba5-a9ce-83cfed37f65d)

I see two 404s in the network tab. One when requesting `https://nextcloud.url/remote.php/dav/files/heyarne/Photos` (the fotos are all in `Pictures`), and one when requesting `https://nextcloud.url/apps/theming/image/background?v=45` (no idea where this is coming from).

Clicking on any one of the grey boxes, the overlay opens but loads indefinitely. There's a DAV PROPFIND request that is being sent, for example to `https://nextcloud.url/remote.php/dav/files/heyarne/Pictures/Found/skateboarding%20in%20the%2080ies.jpg`. There's a status response that seems fine and one for the exif metadata that says it can't be found (if I read the response correctly):

Request:

```xml
<?xml version="1.0"?>
	<d:propfind xmlns:d="DAV:" xmlns:nc="http://nextcloud.org/ns" xmlns:oc="http://owncloud.org/ns" xmlns:ocs="http://open-collaboration-services.org/ns">
		<d:prop>
			<oc:tags />
			<d:getcontentlength /> <d:getcontenttype /> <d:getetag /> <d:getlastmodified /> <d:quota-available-bytes /> <d:resourcetype /> <nc:has-preview /> <nc:is-encrypted /> <nc:mount-type /> <nc:share-attributes /> <oc:comments-unread /> <oc:favorite /> <oc:fileid /> <oc:owner-display-name /> <oc:owner-id /> <oc:permissions /> <oc:share-types /> <oc:size /> <ocs:share-permissions /> <nc:metadata-photos-original_date_time /> <nc:metadata-photos-exif /> <nc:metadata-photos-ifd0 /> <nc:metadata-photos-gps /> <nc:metadata-photos-place />
		</d:prop>
	</d:propfind>
```

Response:

```xml
<?xml version="1.0"?>
<d:multistatus xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns" xmlns:oc="http://owncloud.org/ns" xmlns:nc="http://nextcloud.org/ns"><d:response><d:href>/remote.php/dav/files/heyarne/Pictures/Found/skateboarding%20in%20the%2080ies.jpg</d:href><d:propstat><d:prop><oc:tags/><d:getcontentlength>56960</d:getcontentlength><d:getcontenttype>image/jpeg</d:getcontenttype><d:getetag>&quot;1628f5624e53f6d13cee1a6699442b2f&quot;</d:getetag><d:getlastmodified>Sat, 13 Jun 2020 13:48:32 GMT</d:getlastmodified><d:resourcetype/><nc:has-preview>true</nc:has-preview><nc:mount-type></nc:mount-type><nc:share-attributes>[]</nc:share-attributes><oc:comments-unread>0</oc:comments-unread><oc:favorite>0</oc:favorite><oc:fileid>35</oc:fileid><oc:owner-display-name>heyarne</oc:owner-display-name><oc:owner-id>heyarne</oc:owner-id><oc:permissions>RGDNVW</oc:permissions><oc:share-types/><oc:size>56960</oc:size><x1:share-permissions xmlns:x1="http://open-collaboration-services.org/ns">19</x1:share-permissions><nc:metadata-photos-original_date_time>1592056112</nc:metadata-photos-original_date_time></d:prop><d:status>HTTP/1.1 200 OK</d:status></d:propstat><d:propstat><d:prop><d:quota-available-bytes/><nc:is-encrypted/><nc:metadata-photos-exif/><nc:metadata-photos-ifd0/><nc:metadata-photos-gps/><nc:metadata-photos-place/></d:prop><d:status>HTTP/1.1 404 Not Found</d:status></d:propstat></d:response></d:multistatus>
```

When clicking on the download button in the overlay, the following messages appear in the browser console:

```
File info for /Pictures/Found/skateboarding in the 80ies.jpg fetched 
Object { filename: "/Pictures/Found/skateboarding in the 80ies.jpg", basename: "skateboarding in the 80ies.jpg", lastmod: "Sat, 13 Jun 2020 13:48:32 GMT", size: 56960, type: "file", etag: "1628f5624e53f6d13cee1a6699442b2f", mime: "image/jpeg", tags: "", getcontentlength: 56960, getcontenttype: "image/jpeg", … }
Viewer.vue:631
Could not open file /Pictures/Found/skateboarding in the 80ies.jpg TypeError: e is undefined
    G fileUtils.ts:88
    K file.js:35
    openFileInfo Viewer.vue:725
    openFile Viewer.vue:632
    file Viewer.vue:466
    VueJS 11
    open Viewer.js:301
    openViewer UnassignedFaces.vue:203
    VueJS 4
    emitClick File.vue:183
    click File.vue:1
    VueJS 33
Viewer.vue:639
```

Any idea why the photos are not loaded? The assigned faces work fine!

**Comment by dodog** [comment]
I have the same problem.

**Comment by esterlinpol** [comment]
It is also happening to me.

**Comment by rwolf2467** [comment]
Same Problem here.
Console says: `WebSocket connection to 'wss://nextcloudpi/push/ws' failed.` But "nextcloudpi" is not correct?

**Comment by Liziloo** [comment]
I have the same problem.

**Comment by idkjustgoogleit** [comment]
same here also on newest versions, but with nextcloud hub container from linuxserver,

**Comment by zuktoor2** [comment]
same here... any resolution?
![Screenshot 2025-06-12 160641](https://github.com/user-attachments/assets/8e1f58c0-7b29-4bf1-9057-85f9b9950b8a)

**Comment by mleimbach-hyghstreet** [comment]
seems to be a bug.
go thte smae problem and was searching for a solution and found this discussion.

in my case the vue.js app seems to crash on the result the server sends.
Can this be escalated as a bug ?

**Comment by marcelklehr** [comment]
We are aware of this issue. If someone would like to work on it by contributing a fix to the photos app this would be much appreciated. We are currently not prioritizing work on recognize because of other obligations and the impending rewrite in python. This will hopefully change in the future.

**Comment by idkjustgoogleit** [comment]
I cleared and reset everything via the cli and a recrawl helped.

The people picker under the default nextcloud photo menu doesn't work with memories and or shows different preferred faces of the persons. Opening a person doesn't show anything but memories works.

For now...

-------- Original Message --------
